scripts/build_demand_profiles.py

```diff
@@ -62,6 +62,36 @@ def normed(s):
     return s / s.sum()
 
 
+def get_gegis_regions(countries):
+    """
+    Get the GEGIS region from the config file.
+
+    Parameters
+    ----------
+    region : str
+        The region of the bus
+
+    Returns
+    -------
+    str
+        The GEGIS region
+    """
+    gegis_dict, world_iso = read_osm_config("gegis_regions", "world_iso")
+
+    regions = []
+
+    for d_region in [gegis_dict, world_iso]:
+        for key, value in d_region.items():
+            # ignore if the key is already in the regions list
+            if key in regions:
+                continue
+            # if a country is in the regions values, then load it
+            cintersect = set(countries).intersection(set(value.keys()))
+            if len(cintersect) > 0:
+                regions.append(key)
```

Comment:
Would this fragment get slightly more conscience if:
1) `if not key in regions` conditions would be used to make some actions instead of `if key in regions -> continue`;
2) `if len(cintersect) > 0` would be replaced with `if cintersect > 0`?

Reply:
Thanks!
1. completely agree, done
2. at first, I preferred a more "stable" solution, but "if cintersect" is equivalent. Done :)